src/bot.ts

```diff
@@ -29,18 +30,6 @@ export const identifyPayload: IdentifyPayload = {
   shard: [0, 0],
 };
 
-export interface IdentifyPayload {
```

Comment:
Bring this back and mark it deprecated to make this pr non-breaking

Reply:
@Skillz4Killz the types are not supposed to be private, hence not supposed to be used by the users, but internally.

Reply:
That doesnt mean its not part of the public api. Realistically, you correct. Technically, this is still breaking.

Reply:
> That doesnt mean its not part of the public api. Realistically, you correct. Technically, this is still breaking.

I don't really mind, but just putting that out there...